.coveragerc

```diff
@@ -687,11 +687,6 @@ omit =
     homeassistant/components/konnected/__init__.py
     homeassistant/components/konnected/panel.py
     homeassistant/components/konnected/switch.py
-    homeassistant/components/kostal_plenticore/__init__.py
-    homeassistant/components/kostal_plenticore/helper.py
-    homeassistant/components/kostal_plenticore/select.py
-    homeassistant/components/kostal_plenticore/sensor.py
-    homeassistant/components/kostal_plenticore/switch.py
```

Comment:
Coverage isn't perfect, but they are all > 50% and I think it's worth removing from the ignore list.
```
---------- coverage: platform linux, python 3.12.2-final-0 -----------
Name                                                        Stmts   Miss  Cover   Missing
-----------------------------------------------------------------------------------------
homeassistant/components/kostal_plenticore/__init__.py         26      3    88%   26, 43-44
homeassistant/components/kostal_plenticore/config_flow.py      39      0   100%
homeassistant/components/kostal_plenticore/const.py             1      0   100%
homeassistant/components/kostal_plenticore/coordinator.py     137     39    72%   64-71, 115-116, 138-144, 149, 157-158, 207-215, 234, 273, 292-297, 304-316
homeassistant/components/kostal_plenticore/diagnostics.py      19      0   100%
homeassistant/components/kostal_plenticore/helper.py           60     26    57%   55-56, 64-67, 70-71, 76-79, 84-87, 92-97, 102-107, 116
homeassistant/components/kostal_plenticore/number.py           76      1    99%   202
homeassistant/components/kostal_plenticore/select.py           66      9    86%   137-144, 149-152
homeassistant/components/kostal_plenticore/sensor.py           63     19    70%   827, 856-868, 873, 882-883, 889-890, 895-901
homeassistant/components/kostal_plenticore/switch.py           76     30    61%   81, 111-125, 130, 139-140, 146-147, 151-155, 159-165, 170-176
-----------------------------------------------------------------------------------------
TOTAL                                                         563    127    77%
```

Reply:
The current general rule of 👍 Only remove the ones that are above our project average coverage (Personally: I don't agree with that rule, but that is what it is at this point).

Reply:
OK 👍 
I added them back in, and also added `coordinator.py` even though it's covered at 72%